packages/mermaid/src/diagrams/gantt/parser/gantt.spec.js

```diff
@@ -37,6 +37,12 @@ describe('when parsing a gantt diagram it', function () {
 
     expect(parserFnConstructor(str)).not.toThrow();
   });
+  it('should handle a compact definition', function () {
+    const str =
+      'gantt\ndateFormat yyyy-mm-dd\ntitle Adding gantt diagram functionality to mermaid\ncompact\nexcludes weekdays 2019-02-01';
+
```

Comment:
Thanks for adding tests!

Reply:
Switched to dispalyMode yaml definition

Reply:
I removed this test again, as the current implementation is not parsed by the jison parser. Is there a better place to put parsing tests for YAML/do we test for YAML parsing?